cupy/linalg/einsum.py

```diff
@@ -1,13 +1,15 @@
 import collections
-import re
-import string
 
 import numpy
 import six
 
 import cupy
 
 
+einsum_symbols = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
```

Comment:
`einsum_symbols` should be `'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'` because `A` < `a`.
```
In [1]: import numpy as np

In [2]: np.einsum('abAB', np.ones((2,3,4,5))).shape
Out[2]: (4, 5, 2, 3)

In [3]: np.__version__
Out[3]: '1.14.1'
```